cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -2214,6 +2214,110 @@
+              "label": "<arbitrary_name>",
+              "type": "object",
+              "oneOf": [
+                {
+                  "additionalProperties": false,
+                  "properties": {
+                    "additionalProperties": false,
+                    "type": {
+                      "type": "string",
+                      "enum": ["log"],
+                      "required": true
+                    },
+                    "level": {
+                      "type": "string",
+                      "enum": ["DEBUG", "INFO", "WARN", "ERROR", "FATAL"],
+                      "default": "DEBUG",
+                      "required": false
+                    }
+                  }
+                },
+                {
+                  "additionalProperties": false,
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "enum": ["print"],
+                      "required": true
+                    }
+                  }
+                },
+                {
+                  "additionalProperties": false,
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "enum": ["webhook"],
+                      "required": true
+                    },
+                    "endpoint": {
+                      "type": "string",
+                      "format": "uri",
+                      "description": "The URL to send the event to.",
+                      "required": true
+                    },
+                    "consumer_key": {
+                      "type": "string",
+                      "description": "The consumer key to use for the webhook."
+                    },
+                    "token_key": {
+                      "type": "string",
+                      "description": "The token key to use for the webhook."
+                    },
+                    "token_secret": {
+                      "type": "string",
+                      "description": "The token secret to use for the webhook."
+                    },
+                    "consumer_secret": {
+                      "type": "string",
+                      "description": "The consumer secret to use for the webhook."
+                    },
+                    "timeout": {
+                      "type": "number",
+                      "description": "The timeout in seconds to wait for a response from the webhook."
+                    },
+                    "retries": {
+                      "type": "number",
+                      "description": "The number of times to retry sending the webhook."
+                    }
+                  }
+                },
+                {
+                  "additionalProperties": false,
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "enum": ["hyperv"],
```

Comment:
Would not be useful to document this for the user?

Reply:
I brought it up in our Azure meeting this week. This isn't really intended to be user configurable and gets enabled automatically in `/etc/cloud` on Azure instances. I asked if they wanted this to be more broadly documented at our last sync up and they said they'd get back to me. For now, let's leave it.